Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -59,11 +60,201 @@
 #import <WebCore/ResourceResponse.h>
 #import <WebCore/ScrollAnimator.h>
 #import <WebCore/SharedBuffer.h>
+#import <WebCore/WebAccessibilityObjectWrapperMac.h>
 #import <wtf/cocoa/VectorCocoa.h>
 #import <wtf/text/TextStream.h>
-
 #import "PDFKitSoftLink.h"
 
+
+@interface WKAccessibilityPDFDocumentObject: NSObject {
+    RetainPtr<PDFDocument> _pdfDocument;
+    WeakObjCPtr<NSObject> _parent;
+}
+
+@property (assign) WeakPtr<WebCore::HTMLPlugInElement, WebCore::WeakPtrImplWithEventTargetData> pluginElement;
+@property (assign) WebKit::PDFPluginBase* pdfPlugin;
+
+- (id)initWithPDFDocument:(RetainPtr<PDFDocument>)document andElement:(WebCore::HTMLPlugInElement *)element;
+- (void)setPDFPluginBase:(WebKit::PDFPluginBase *)plugin;
+- (void)setPDFDocument:(RetainPtr<PDFDocument>)document;
+- (void)setParent:(NSObject *)parent;
+- (id)accessibilityHitTest:(NSPoint)point;
+- (PDFDocument*)document;
```

Comment:
Star on the left for C++, right for ObjC, this is ObjC so you're missing a space.